src/main/java/be/seeseemelk/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -11,11 +11,7 @@
 import java.util.Set;
 import java.util.UUID;
 
-import org.bukkit.Bukkit;
-import org.bukkit.FluidCollisionMode;
-import org.bukkit.GameRule;
-import org.bukkit.Location;
-import org.bukkit.Material;
+import org.bukkit.*;
```

Comment:
Never use wildcard imports please.

Reply:
Yes sorry, I'll disable this feature in Intellij